code/modules/shuttle/shuttle.dm

```diff
@@ -517,7 +517,7 @@
 
 			//move mobile to new location
 			for(var/atom/movable/AM in T0)
-				if(AM.onShuttleMove(T1, rotation))
+				if(AM.onShuttleMove(T1, rotation, force))
```

Comment:
potential change in functionality, make this argument source its value from the docking port object, and set it to false on your elevators.
eg:
```
/obj/docking_port/elevator
    knockdown = FALSE
```